### PR TITLE
Remove authentication in bulk translation downloads

### LINKED ISSUE #1991: Avoid user/password login for GlotPress bulk download
## Problem

Logging in with the username and password for bulk downloads from GlotPress causes lots of logins from my personal account to WordPress.org.

## Description

We should restrict that to specifically to frontend deploys and even there, possibly switch to a secret-based approach.

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes #1991 by @dhruvkb
Reported by @dd32

## Description

This PR removes the username/password authentication for WordPress.org. This does not affect bulk downloads as they can be downloaded without authentication as well.

This PR is accompanied by an infra PR https://github.com/WordPress/openverse-infrastructure/pull/491 to remove the secret from GitHub's secret store.

If everything works well, we can deprecate and remove the rate-limited separate download code from the repo.

## Testing Instructions

- Check out the PR.
- Remove all existing locale JSON files from `frontend/src/locales/`
- Download translations using `just frontend/run i18n:get-translations`
- Even without any env vars or credentials, the translations are downloaded in bulk

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
Bulk download is working well. See usages:

- Build Docker images (frontend): https://github.com/WordPress/openverse/actions/runs/4869253856/jobs/8683582793?pr=1993#step:5:24
- Check Nuxt build: https://github.com/WordPress/openverse/actions/runs/4869253856/jobs/8683582300#step:4:29

**Comment by krysal** [comment]
Aren't bulk downloads throttled without authentication?

File: automations/js/src/last_week_tonight.mjs [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { escapeHtml } from './html.mjs'
 const pat = process.env.ACCESS_TOKEN
 /** the username for the Make site account making the post */
 const username = process.env.MAKE_USERNAME
-/** the application password for the Make site */
+/** the application password, not login password, for the Make site */
 const password = process.env.MAKE_PASSWORD
```
Comment (sarayourfriend) [inline automations/js/src/last_week_tonight.mjs:22]: Why do we still need it for the weekly updates :thinking:

**Review by sarayourfriend — COMMENTED** [review]
Clarifying question before finishing the review.

File: automations/js/src/last_week_tonight.mjs (hunk 1 above)
Comment (dhruvkb) [reply automations/js/src/last_week_tonight.mjs:22]: The weekly updates workflow needs to post to the Make site, which is an authenticated operation.

This job only fetches the translations, and does not need to be authenticated (it used to be, but now it's opened up).

Comment (sarayourfriend) [reply automations/js/src/last_week_tonight.mjs:22]: Ah! Gotcha, I didn't realise this wasn't related to the translations :+1:

**Review by sarayourfriend — APPROVED** [review]
LGTM! Thanks for the quick explanation.

**Pull request merged**